<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="LoginWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>81</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>41</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>82</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>记住配置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TableEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>108</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>数据库名：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginWindow</class>
   <extends>QWidget</extends>
   <header>loginwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
